edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/config/ConfigUpdaterTest.cs

```diff
@@ -86,9 +87,10 @@ public async Task TestPeriodicConfigUpdate()
                 .ReturnsAsync(Option.Some(edgeHubConfig5))
                 .ReturnsAsync(Option.Some(edgeHubConfig6));
             configProvider.Setup(c => c.SetConfigUpdatedCallback(It.IsAny<Func<EdgeHubConfig, Task>>()));
+            var diskSpaceChecker = Mock.Of<IStorageSpaceChecker>();
 
             // Act
-            var configUpdater = new ConfigUpdater(router, messageStore.Object, updateFrequency);
+            var configUpdater = new ConfigUpdater(router, messageStore.Object, updateFrequency, diskSpaceChecker);
```

Comment:
I have tests for this class in my PR, just letting you know so you don't need to write them